uniflow/op/extract/split/markdown_header_splitter.py

```diff
@@ -53,7 +57,8 @@ def __call__(
         for node in nodes:
             value_dict = copy.deepcopy(node.value_dict)
             text = value_dict["text"]
-            text = self.header_splitter(text.strip(), headers_to_split_on_list)
+            self.header_splitter(text.strip(), headers_to_split_on_list)
+            text = self.format_splitter()
```

Comment:
qq: what is the purpose of introducing a `lines_with_metadata` class variable instead of directly passing the header_splitter return value into format_splitter?